package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@t-network/solidity-contracts",
+  "version": "0.0.1",
+  "license": "MIT",
```

Comment:
MIT, GPLv3, ...?

Reply:
G-P-L, G-P-L.

IMO 😏

Reply:
For reference, all current NuCypher contracts are AGPLv3

Reply:
What if we start with GPLv3 and then revisit so that we are not blocking the setup PR? I think AGPLv3 could affect clients having to bundle contract bindings in the source code and binaries. 🤔

Reply:
Sure, let's do that 👍

Reply:
Done in https://github.com/threshold-network/solidity-contracts/pull/2/commits/876570b7f518ec18d231e562dd5972336d00dc91.